lib/state/electron/middleware.ts

```diff
@@ -100,6 +104,12 @@ export const middleware: S.Middleware = ({ dispatch, getState }) => {
     const result = next(action);
     const nextState = getState();
 
+    switch (action.type) {
+      case 'REALLY_CLOSE_WINDOW':
+        console.log('REALLY_CLOSE_WINDOW');
+        window.electron.send('reallyCloseWindow');
+        return result;
+    }
```

Comment:
probably want a newline here. I'm surprised `prettier` didn't add it.

also let's get rid of that console statement

Reply:
Console resolved in: https://github.com/Automattic/simplenote-electron/pull/2277/commits/bb069aed17218e442be9646f4dd46c85201bf610

New line resolved in: https://github.com/Automattic/simplenote-electron/pull/2277/commits/6dfd9843c1cdd0bb802a4fdd174202fa0bdbdc07